src/components/Tabs/TabContainer.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+
+const Div = styled.div`
+  ${({ vertical }) => {
+    if (vertical) {
+      return css`  
+    width: 100%;
+    display: table;
+  `;
+    }
+  }}`;
+
+export default class TabContainer extends React.Component {
+
+  static propTypes = {
+    /**
+     * Tab elements within container
+     */
+    children: PropTypes.node,
+    /**
+     * Enable vertical tab layout
+     */
+    vertical: PropTypes.bool,
+  };
+
+  attachParams = (children, vertical) => {
+    if (typeof children !== 'object') {
```

Comment:
This check is kind of confusing, and I don't think it should be necessary. `React.Children.map` automatically casts children to an array.

Reply:
done